test/regression.uts

```diff
@@ -1825,7 +1825,7 @@ class ATMT5(Automaton):
     @ATMT.condition(BEGIN, prio=-1)
     def tr3(self):
         self.res += "u"
-    
+
```

Comment:
The spacing changes in the automaton classes break them. You could probably delete those spacing lines.